packages/flutter/lib/src/services/platform_channel.dart

```diff
@@ -128,6 +128,12 @@ class MethodChannel {
   /// result. The values supported by the default codec and their platform-specific
   /// counterparts are documented with [StandardMessageCodec].
   ///
+  /// The generic argument `T` of the method can be inferred by the surrounding
+  /// context, or provided explicitly. If it does not match the returned type of
+  /// the channel, a [TypeError] will be thrown at runtime. `T` cannot be class
+  /// with generics, such as `Map<String, String>`, instead it must always
+  /// return an untyped map or list.
```

Comment:
untyped meaning what? `Map<dynamic, dynamic>` isn't "untyped", it's just typed with dynamic...

Reply:
Re-phrased to clarify that only dynamic is supported